docs/en/operations/opentelemetry.md

```diff
@@ -31,6 +31,10 @@ The table must be enabled in the server configuration, see the `opentelemetry_sp
 
 The tags or attributes are saved as two parallel arrays, containing the keys and values. Use [ARRAY JOIN](../sql-reference/statements/select/array-join.md) to work with them.
 
+## Log-query-settings
```

Comment:
@sharathks118 Without a link to the corresponding setting in "settings.md", readers will be confused by "Log-query-settings" and below description.